terraform/deployments/chat/cloudfront.tf

```diff
@@ -113,3 +113,8 @@ resource "aws_cloudfront_function" "add_index" {
   runtime = "cloudfront-js-2.0"
   code    = file("${path.module}/add_index.js")
 }
+
+resource "aws_shield_protection" "chat_shield_protection" {
```

Comment:
This will cause errors if `var.cloudfront_create` is not set to true, since it will be referencing a resource that doesnt exist